api/sft/handler.go

```diff
@@ -27,11 +28,25 @@ func NewHandler(r chi.Router, s *Service) *Handler {
 
 func StartDashboard(port string) {
 
-	// TODO: sort out path to the dashboard
-	fs := http.FileServer(http.Dir("/Users/maxbb/github/revitt/sft/web/dashboard"))
-	http.Handle("/", fs)
+	//TODO: sort out path to the dashboard - this needs to be managed by a config when the dashboard is ready
+	buildPath := "/Users/maxbb/github/revitt/sft/web/dashboard/dist"
+
+	fs := http.FileServer(http.Dir(buildPath))
+
+	// this handler function is designed to allow us to serve a react app with react router
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		// Check if the requested file exists
+		path := buildPath + r.URL.Path
+		if _, err := os.Stat(path); os.IsNotExist(err) {
+			// If the file does not exist, serve index.html
+			http.ServeFile(w, r, buildPath+"/index.html")
+		} else {
+			// Otherwise, serve the file
+			fs.ServeHTTP(w, r)
+		}
+	})
 
-	log.Printf("Starting dashboard server on http://localhost:%s/", port)
+	log.Printf("Starting dashboard server on http://hello:%s/", port)
```

Comment:
Let's user our logger.Log().Level(x) - if we're consistent with this that would be good. That's one thing we need to do - move our wrapped logger out into it's own package at some point!!!

Reply:
I've updated all of the printlns to be logger.Log().Info() - hope this is right.